.eslintrc

```diff
@@ -0,0 +1,11 @@
+env:
+  es6: true
+  mocha: true
+  node: true
+
+extends:
+  - eslint:recommended
+
+rules:
+  no-var: 0
+  prefer-const: 0
```

Comment:
Nit, we should also add `indent: [2, 4]` since there was at least 2-3 whitespace violations.